sklearn/linear_model/_logistic.py

```diff
@@ -721,6 +721,9 @@ def _log_reg_scoring_path(
         else:
             score_params = score_params or {}
             score_params = _check_method_params(X=X, params=score_params, indices=test)
+            # FIXME: If scoring = "neg_brier_score" and if not all class labels
```

Comment:
For a custom scorers, I think the caller is responsible for creating a scorer with labels set, but it's not obvious.